apps/dashboard/app/url.ts

```diff
@@ -0,0 +1,17 @@
+interface UrlInfo {
+  title: string;
+  protected: boolean;
+}
+
+const URLS: Record<string, UrlInfo> = {
+  "/": { title: "Home", protected: true },
+  "/transaction": { title: "Transaction", protected: true },
+};
+
+export function getTitle(path: string): string {
```

Comment:
what is the purpose of this function?

Reply:
To extract the title from a URL, adding it in the component will make it easier to understand.